README.md

```diff
@@ -130,7 +114,7 @@ and to [avoid insecure fallback in go
 get](https://github.com/golang/go/issues/9637).
 
 Local development also requires a RabbitMQ installation and MariaDB
-10 installation. MariaDB should be run on port 3306 for the
+10 or MySQL installation (see above). MariaDB should be run on port 3306 for the
```

Comment:
Agreed we can't support both. I was thinking it would be worthwhile to say "MySQL will prooobably work if that's what you've got." But yeah thinking about it again, there's no reason to say that in the readme, that will just imply we support it. Will remove.